Assets/HoloToolkit/Utilities/Scripts/Editor/SceneSettingsWindow.cs

```diff
@@ -95,7 +97,23 @@ protected override void ApplySettings()
                 }
 
                 PrefabUtility.InstantiatePrefab(AssetDatabase.LoadAssetAtPath<GameObject>(AssetDatabase.GUIDToAssetPath(InputSystemPrefabGUID)));
-                FocusManager.Instance.UpdateCanvasEventSystems();
+                Values[SceneSetting.UpdateCanvases] = true;
```

Comment:
Is this required for adding the input system prefab, or should we give the choice of one or the other instead of forcing it even if I don't check the Update Canvases box?

Reply:
This is required when using the input system.

Reply:
New setups are auto opt in, while existing setups are opt out.